content/pt-br/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

```diff
@@ -0,0 +1,205 @@
+   Visualize o status do conjunto de réplicas usando:
+
+   ```shell
+   kubectl get replicaset
+   ```
+
+   Visualize o status do conjunto de réplicas usando:
+
+   ```
+   NAME               DESIRED   CURRENT   READY   AGE
+   mongo-75f59d57f4   1         1         1       3m12s
+   ```
+
+2. Crie um serviço (*Service*) para expor o MongoDB na rede:
+
+   ```shell
+   kubectl apply -f https://k8s.io/examples/application/mongodb/mongo-service.yaml
+   ```
+
+   A saída (output) de um comando bem-sucedido verifica que o serviço foi criado:
+
+   ```
+   service/mongo criado
+   ```
+
+   Verifique o serviço criado::
+
+   ```shell
+   kubectl get service mongo
+   ```
+
+   A saída exibe o serviço criado:
+
+   ```
+   NAME    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
+   mongo   ClusterIP   10.96.41.183   <none>        27017/TCP   11s
+   ```
+
+3. Verifique se o servidor MongoDB está sendo executado no Pod e ouvindo a porta 27017:
+
+   ```shell
+   # Altere mongo-75f59d57f4-4nd6q para o nome do Pod
+   kubectl get pod mongo-75f59d57f4-4nd6q --template='{{(index (index .spec.containers 0).ports 0).containerPort}}{{"\n"}}'
+   ```
+
+   A saída exibe a porta para o MongoDB nesse Pod:
+
+   ```
+   27017
+   ```
+
+   27017 é a porta TCP alocada ao MongoDB na internet.
+
+## Encaminhe uma porta local para uma porta no Pod
+
+1. `kubectl port-forward` permite usar o nome do recurso, como o nome do pod, para selecionar um pod correspondente para encaminhar a porta.
+
+
+   ```shell
+   # Altere mongo-75f59d57f4-4nd6q para o nome do Pod
+   kubectl port-forward mongo-75f59d57f4-4nd6q 28015:27017
+   ```
+
+   que é o mesmo que
+
+   ```shell
+   kubectl port-forward pods/mongo-75f59d57f4-4nd6q 28015:27017
+   ```
+
+   ou
+
+   ```shell
+   kubectl port-forward deployment/mongo 28015:27017
+   ```
+
+   ou
+
+   ```shell
+   kubectl port-forward replicaset/mongo-75f59d57f4 28015:27017
+   ```
+
+   ou
+
+   ```shell
+   kubectl port-forward service/mongo 28015:27017
+   ```
+
+   Qualquer um dos comandos acima funciona. A saída é semelhante a esta:
+
+   ```
+   Encaminhamento de 127.0.0.1:28015 -> 27017
+   Encaminhamento de [::1]:28015 -> 27017
+   ```
+
+   {{< note >}}
+   `kubectl port-forward` não retorna. Para continuar com os exercícios, você precisará abrir outro terminal.
+   {{< /note >}}
+
+2. Inicie a interface de linha de comando do MongoDB:
+
+   ```shell
+   mongosh --port 28015
+   ```
+
+3. No prompt de comando do MongoDB, digite o comando `ping`:
+
+   ```
+   db.runCommand( { ping: 1 } )
+   ```
+
+   Uma solicitação de ping bem-sucedida retorna:
+
+   ```
+   { ok: 1 }
+   ```
+
+### Opcionalmente, deixe kubectl escolher a porta local {#let-kubectl-choose-local-port}
+
+Se você não precisa de uma porta local específica, pode permitir que o `kubectl` escolha e aloque a porta local e, assim, se livrar da necessidade de gerenciar conflitos de porta local, com a sintaxe ligeiramente mais simples:
```

Comment:
Can we change aloque to reserve and maybe "se livrar" to something that uses the word evitar?